ios/swift/AmpliSwiftSampleApp/Shared/TextView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TextView.swift
+//  AmpliSwiftSampleApp
+//
+//  Created by Qingzhuo Zhen on 12/1/21.
+//
+
+import SwiftUI
+
+struct TextView: View {
+    var body: some View {
+        //    'Ampli.instance' is the default instance of Ampli()
+        //
+        //    When you pull your tracking plan you can use the defaults and call load() without arguments
+        //    This requires connecting your account via `ampli pull` which will set you API key in the generated Ampli SDK
+        //
+        //    OR Specify a AmpliEnvironment
+        //    ampli.load(options: LoadOptions(environment: AmpliEnvironment.development)
+        //
+        //    OR Provide a specific Amplitude API key
+        //    ampli.load(options: LoadOptions(client: LoadClientOptions(apiKey: "Custom api key"))
+        //
+        //    OR Use an existing Amplitude instance
+        //    requires "import Amplitude"
+        //    let instance = Amplitude.instance("instanceName");
+        //    instance.initializeApiKey("Custom api key");
+        //    ampli.load(options: LoadOptions(client: LoadClientOptions(instance: instance)))
+        //
+        //    For testing you can disable ampli
+        //    ampli.load(options: LoadOptions(disabled: ENV.IS_TESTING ? true: false))
+        //
+        //    Make as many Ampli instances as you want
+        //    let ampli2 = new Ampli();
+        //    ampli2.load(options: LoadOptions(client: LoadClientOptions(apiKey: "api-key-2")))
+        let apiKey = ProcessInfo.processInfo.environment["AMPLITUDE_API_KEY"];
+        let ampli = Ampli.instance
+        ampli.load(options: LoadOptions(client: LoadClientOptions(apiKey: apiKey)));
```

Comment:
`Ampli.instance.load()` should be call once at app start up. Should we move it out of the TextView and into AmpliSwiftSampleAppApp? We used to call itly.load() in AppDelegate in the old Itly examples.

I am thinking we can move most of this to `AmpliSwiftSampleAppApp` and then just make one or two event calls here to show how to use the `Ampli.instance` in multiple files.

Reply:
@justin-fiedler Recently there are some issues reported about session related to initialize on App load inside AppDelegate. I am wondering if we should set that up inside our example, although session is not used here. Some customers wrote a good explanation about it in https://community.amplitude.com/instrumentation-and-data-management-57/ios-silent-notification-problem-how-they-mess-up-your-app-s-lifecycle-and-amplitude-session-events-492.

Reply:
Interesting, thanks for the article. I think to keep the example simple could still do `ampli.load()` in the App to make it clear that it should only happen once. It seems this is also what we recommend in the current Amplitude-iOS docs here - https://developers.amplitude.com/docs/ios#initialization

Hopefully anyone with session issues can look up the more advanced usage in the article you provided.

Reply:
OK. Just updated and moved to AmpliSwiftSampleAppApp for now. Just realized in Ampli we don't enable session for ios by default though.

Reply:
For now let's keep the Ampli interface minimal. Users can either configure their own instance or set via `Ampli.instance.client` 

Maybe this is something we can add to LoadOptions in the future, and would be made easier if the v2 core Sdk exported a configuration interface that could be used instead of individual methods. e.g.

```
AmpConfiguration {
  serverZone: AMPServerZone,
  trackingSessionEvents: boolean;
}

Amplitude.instance().init(serverZone: AMPServerZone.EU, trackingSessionEvents: true)
```

Reply:
Sounds good. A configuration interface would be more ideal for future.